cai_causal_graph/causal_graph.py

```diff
@@ -473,12 +474,27 @@ def __init__(
 
         # Add nodes.
         if input_list is not None:
+            warnings.warn(
+                'Passing input_list during construction of a CausalGraph is now deprecated, and will be '
+                'removed in future versions. Please add nodes and edges after construction instead.',
```

Comment:
Would suggest to change "Please add nodes and edges after construction instead." to "Please add nodes via the add_node method and edges via the add_edge method instead."

Just to be a bit more specific. What do you think?

Reply:
Resolved offline, pointing to add_fully_connected_nodes instead.